staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -680,8 +680,48 @@ func TestWildcardMasking(t *testing.T) {
 					},
 				},
 			},
-			expectedError: true,
-			errorString:   fmt.Sprintf("resource secrets is masked by earlier rule %s", schema.GroupResource{Group: "", Resource: "*"}),
+			expectedError: fmt.Errorf("resource secrets is masked by earlier rule %s", schema.GroupResource{Group: "", Resource: "*"}),
```

Comment:
Per https://github.com/kubernetes/kubernetes/pull/115149#discussion_r1121954350 - write out the raw error string with no use of any `fmt` function - i.e. do not use `%` formatting.